docs/examples/pp_greedy.rst

```diff
@@ -0,0 +1,129 @@
+.. _pp_greedy:
+
+===================================
+Postprocessing with a Greedy Solver
+===================================
+
+This example uses explicit postprocessing to improve results returned 
+from a quantum computer. 
+
+Typically, Ocean tools do some minimal, implicit postprocessing; for 
+example, when you use 
+`embedding tools <https://docs.ocean.dwavesys.com/en/stable/docs_system/reference/embedding.html>`_
+to map problem variables to qubits, *broken* :term:`chain`\ s (differing
+states of the qubits representing a variable) may be resolved by majority 
+vote: Ocean sets the variable's value based on the state returned from 
```

Comment:
seems like a good opportunity to link to majority vote function

Reply:
I linked to the page a bit earlier in the example: 
```
when you use 
`embedding tools <https://docs.ocean.dwavesys.com/en/stable/docs_system/reference/embedding.html>`_
to map problem variables to qubits, *broken* :term:`chain`\ s (differing states of the qubits representing a variable) may be resolved by majority vote
```
I'm chary of linking to anything lower level that might move and break link